turbo.json

```
@@ -36,7 +36,20 @@
       "outputs": ["src/gen/**"]
     },
     "test": {
-      "dependsOn": ["build", "^test", "test:integration", "test:api"]
+      "dependsOn": [
+        "build",
+        "^test",
+        "^test:old",
+        "test:integration",
+        "test:integration:old",
+        "test:api"
+      ]
+    },
+    "test:old": {
+      "dependsOn": ["build", "^test:old", "test:integration:old"]
```

Dependency not needed?

```suggestion
      "dependsOn": ["build", "^test:old"]
```